opencti-platform/opencti-front/src/private/components/cases/tasks/CaseTaskCreation.tsx

```diff
@@ -153,25 +167,36 @@ const CaseTaskCreation: FunctionComponent<CaseTaskCreationProps> = ({
           />
           <ObjectAssigneeField
             name="objectAssignee"
+            required={(mandatoryAttributes.includes('objectAssignee'))}
             style={fieldSpacingContainerStyle}
           />
           <ObjectParticipantField
             name="objectParticipant"
+            required={(mandatoryAttributes.includes('objectParticipant'))}
             style={fieldSpacingContainerStyle}
           />
+          <CreatedByField
```

Comment:
Please remove this field, we don't want to allow an author to be selected for a task.

Reply:
@frapuks - Author is a configurable field off of the Customize. It was not present on the form for create, but present on the form for Edit. It was added here to the form, because if you use the customize and make it required, with it not on the form - no task can be created anymore. Should it be defaulted to some value (if required, but not on the form?)

Reply:
![image](https://github.com/user-attachments/assets/a63766e5-0d86-460a-b898-45d532a9ab23)